packages/util/src/constants.ts

```diff
@@ -141,3 +152,72 @@ export const ACCOUNT_ID_RANGES = [
   '0xf000000000000000000000000000000000000000',
   '0xffffffffffffffffffffffffffffffffffffffff',
 ];
+
+// all addresses must be properly checksummed
+export const tokens = {
+  [Network.mainnet]: {
+    WETH: '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2',
+    GHO: '0x40D16FC0246aD3160Ccc09B8D0D3A2cD28aE6C2f',
+    rETH: '0xae78736Cd615f374D3085123A210448E74Fc6393',
+    PYUSD: '0x6c3ea9036406852006290770BEdFcAbA0e23A0e8',
+    USDC: '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48',
+    USDT: '0xdAC17F958D2ee523a2206206994597C13D831ec7',
+    crvUSD: '0xf939E0A03FB07F59A73314E73794Be0E57ac1b4E',
+    CRV: '0xD533a949740bb3306d119CC777fa900bA034cd52',
+    CVX: '0x4e3FBD56CD56c3e72c1403e103b45Db9da5B9D2B',
+    BAL: '0xba100000625a3754423978a60c9317c58a424e3D',
+    AURA: '0xC0c293ce456fF0ED870ADd98a0828Dd4d2903DBF',
+    WBTC: '0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599',
+  },
+  [Network.arbitrum]: {
+    WETH: '0x82aF49447D8a07e3bd95BD0d56f35241523fBab1',
+    BAL: '0x040d1EdC9569d4Bab2D15287Dc5A4F10F56a56B8',
+    AURA: '0x1509706a6c66CA549ff0cB464de88231DDBe213B',
+    CRV: '0x11cDb42B0EB46D95f990BeDD4695A6e3fA034978',
+    ARB: '0x912CE59144191C1204E64559FE8253a0e49E6548',
+    USDCe: '0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8',
+    USDC: '0xaf88d065e77c8cC2239327C5EDb3A432268e5831',
+    USDT: '0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9',
+    wstETH: '0x5979D7b546E38E414F7E9822514be443A4800529',
+    WBTC: '0x2f2a2543B76A4166549F7aaB2e75Bef0aefC5B0f',
+    FRAX: '0x17FC002b466eEc40DaE837Fc4bE5c67993ddBd6F',
+  },
+};
+
+export const vaults = {
```

Comment:
We also have this function in core-entities called `whitelistedVaults` we should probably combine with this data structure somehow